buildSrc/build.gradle.kts

```
@@ -55,12 +55,30 @@ repositories {
     mavenLocal()
     gradlePluginPortal()
     mavenCentral()
+    maven {
+        url = uri("https://repo.spring.io/libs-release")
+    }
```

This should also work:
```suggestion
    maven("https://repo.spring.io/libs-release")
```

Also, maybe worth adding a note on why do we need Spring repos here?

---

@alexander-yevsyukov `buildSrc` isn't completely "copied". The `build.gradle.kts` belongs to this project.

---

@armiol, can you shed some light on Spring matter, please? I also had this question, but was lazy enough to get down to exploring it under `config`. Now, after Yuri's comments I'm there... :)

After your answer I'll update `config` accordingly.

---

@armiol, do we need Spring Maven repo for this dependency `org.ajoberstar.grgit.Grgit`?

 They specify another Maven repo in their project page (https://github.com/ajoberstar/grgit).

The problem is that I don't see `ajoberstar` at Spring Maven repo:
https://repo.spring.io/ui/native/libs-release/org

---

https://ajoberstar.org/bintray-backup/ says it's only for older versions of the tool.